h/views/accounts.py

```diff
@@ -142,18 +142,12 @@ def _login_redirect(self):
         return self.request.params.get('next', _login_redirect_url(self.request))
 
     def _login(self, user):
-        # Clear any cached feature flags
-        self.request.feature.clear()
```

Comment:
Ah nice. I first thought that we still need those in case we would access the features before the `authenticated_user` is set. But you changed the service to evaluate the feature state on every call by caching the feature flags, and not the state of the flag for the current user 👍